packages/cli/__tests__/zosfiles/__system__/delete/zfs/cli.files.delete.zfs.system.test.ts

```diff
@@ -114,6 +115,20 @@ describe("Delete z/OS File System", () => {
             expect(response.status).toBe(0);
             expect(response.stdout.toString()).toMatchSnapshot();
         });
+
+        it("should delete a ZFS with --ignore-not-found flag", async () => {
+            // first create zfs
+            const createResponse = runCliScript(__dirname + "/__scripts__/command/command_create_zfs.sh",
+                TEST_ENVIRONMENT, [fsname, volume]);
+            expect(createResponse.status).toBe(0);
+
+            const deleteResponse = runCliScript(__dirname + "/__scripts__/command/command_delete_zfs.sh",
+                TEST_ENVIRONMENT, [fsname, "--for-sure", "--ignore-not-found"]);
```

Comment:
Same question as for USS above - should we attempt the deletion a 2nd time and make sure it still works with `--inf` flag?